docs/get-started/README.md

```diff
@@ -1,73 +1,85 @@
-  <img width={700} src="https://cloud.githubusercontent.com/assets/4991911028165/49f41da2-86bc-11e5-85eb-9279621ef971.png" />
-</div>
-
-react-map-gl is a [React](http://facebook.github.io/react/)-friendly
-wrapper for [MapboxGL-js](https://www.mapbox.com/mapbox-gl-js/), a WebGL-powered
-vector and raster tile mapping library.
-
-MapboxGL-js provides impressive vector tile rendering capabilities
-that you can find out more about [here](https://www.mapbox.com/mapbox-gl-js/).
-Although the Mapbox
-[vector tile specification](https://www.mapbox.com/developers/vector-tiles/) is 
-[open source](https://github.com/mapbox/vector-tile-spec), there aren't yet very
-many free alternatives to Mapbox's paid 
-[vector tile API](https://www.mapbox.com/pricing/). Because of this, the
-examples here don't use the paid vector tile API and instead use map tiles by
-[Stamen Design](http://stamen.com), under
-[CC BY 3.0](http://creativecommons.org/licenses/by/3.0) and data by
-[OpenStreetMap](http://openstreetmap.org),
-under [ODbL](http://www.openstreetmap.org/copyright).
-
-Two guiding principles this library tries to adhere to:
-
-## Stateless
-
-One of the goals of the project is to provide a [stateless](https://facebook.github.io/react/docs/interactivity-and-dynamic-uis.html#what-components-should-have-state) API to the underlying map and overlay components. (Also see [Pure UI](http://rauchg.com/2015/pure-ui/) by [Guillermo Rauch](https://twitter.com/rauchg).)
-
-## Small modules
+# Installation
 
-[Small modules]() - Whenever possible, isolated, reusable code should exist in separate npm modules that [do one thing well](https://en.wikipedia.org/wiki/Unix_philosophy#Do_One_Thing_and_Do_It_Well). (Also see [Module Best Practices](https://github.com/mattdesl/module-best-practices) by [Matt DesLauriers](https://twitter.com/mattdesl).)
+`ReactMapGL` requires `node >= v4` and `react >= 15.4`.
 
+```sh
+npm install --save react-map-gl
+```
+or
+```sh
+yarn add react-map-gl
+```
 
-# Installation
+## Example
+
+```js
+import {Component} from 'react';
+import ReactMapGL from 'react-map-gl';
+
+class Map extends Component {
+  render() {
+    return (
+      <ReactMapGL
+        width={400}
+        height={400}
+        latitude={37.7577}
+        longitude={-122.4376}
+        zoom={8}
+        onViewportChange={(viewport) => {
+          const {latitude, longitude, zoom} = viewport;
+          // Optionally call `setState` and use the state to update the map.
+        }}
+      />
+    );
+  }
+}
+```
 
-react-map-gl needs to be installed in a product that also requires react 0.14.x and ImmutableJS 3.x. (These are referred to as its [peerDependencies](https://nodejs.org/en/blog/npm/peer-dependencies/)) so if you haven't already, you'll need to run the following in your project:
+## Server Side Rendering
 
-    npm install react
-    npm install immutable
-    npm install r-dom
+`ReactMapGL` depends on `gl`, which may cause issues when running in a server
```

Comment:
+1 for documenting this.

Idea: Given how much people struggle with this, I thought we could supply a prop to control whether mapbox is loaded under Node.js, default to `false`.